app/models/race_entry.rb

```diff
@@ -11,6 +13,8 @@ class RaceEntry < ActiveRecord::Base
 
   after_initialize :default_values
 
+  attr_accessor :category_name
```

Comment:
This is just a handy way to add a "virtual attribute" that allows us to set and get a value for each record without saving it to the database.